uaclient/actions.py

```diff
@@ -285,3 +292,30 @@ def collect_logs(cfg: config.UAConfig, output_dir: str):
             system.write_file(
                 os.path.join(output_dir, os.path.basename(f)), content
             )
+
+    # get apparmor logs
+    # can't use journalctl's --grep, because xenial doesn't support it :/
+    kernel_logs, _ = system.subp(
```

Comment:
Good point. What are my options here if this call fails? Since `pro collect-logs` will most likely be run by a user, I think the error should be displayed in the terminal, and not (just?) `ubuntu-ardantage.log`.

Reply:
or should I just use `LOG.warning()`? I see this pattern in other places in collect-logs

Reply:
Tried the latter, let's see.

Reply:
I think logging works as well and maybe we can also add the same approach as `_write_command_output_to_file` where we write a file with an "-error" suffix containing the exception raised

Just for the case where the user ignores the warning and upload the tarball anyway to the bug.

What do you think ?

Reply:
Won't the warning (`LOG.warning`)be in the logs that are being uploaded? But sure, I can add another file

Reply:
Yes, the logs should be present in the main log files. The file idea is just for us to easily identify that the apparmor profiles were not collected